overlord/snapstate/check_snap_test.go

```diff
@@ -1058,24 +1042,21 @@ func (s *checkSnapSuite) TestCheckSnapSystemUsernames(c *C) {
 		defer restore()
 
 		release.OnClassic = test.classic
```

Comment:
(nitpick) could we use "restore := release.MockOnClassic(test.classic); defer restore()" here and remove it from the top of the test?

Reply:
Sure. Note that this follows the convention in TestCheckSnapAssumes() up above.

Reply:
Done